src/components/signature-area/signature-area.ts

```diff
@@ -35,6 +35,12 @@ export class SignatureAreaComponent implements ControlValueAccessor {
   @Input()
   public signHereText: string;
 
+  @Input()
+  public isValidText: string;
```

Comment:
I'd change the name of this `@Input` to something along the lines of `validationErrorText`.